en/advanced_config/sensor_thermal_calibration.md

```diff
@@ -12,94 +14,104 @@ The inertial rate gyro and accelerometer sensor offsets are calculated using a 3
 
 ![Thermal calibration barometer](../../assets/calibration/thermal_calibration_baro.png)
 
-## Calibration Parameter Storage
-
-With the existing parameter system implementation we are limited to storing each value in the struct as a separate entry. To work around this limitation the following logical naming convention for the parameters is used:
+### Calibration Parameter Storage
 
-TC\_[type][instance]\_[cal\_name]\_[axis] , where
+With the existing parameter system implementation we are limited to storing each value in the struct as a separate entry. To work around this limitation the following logical naming convention is used for the [thermal compensation parameters](../advanced_config/parameter_reference.md#thermal-compensation):
```

Comment:
"we are limited to storing each value in the struct as a separate entry." - What struct are we talking about here - has not previously been mentioned.